packages/dashboard/src/components/ConfirmDialog/index.tsx

```diff
@@ -0,0 +1,102 @@
+import { MaskColorVar } from '@dimensiondev/maskbook-theme'
+import {
+    Dialog,
+    DialogTitle,
+    DialogContent,
+    DialogActions,
+    Button,
+    Typography,
+    IconButton,
+    dialogActionsClasses,
+    dialogContentClasses,
+    experimentalStyled as styled,
+    buttonClasses,
+} from '@material-ui/core'
+import CloseIcon from '@material-ui/icons/Close'
+
+export interface DialogTitleProps {
+    children?: React.ReactNode
+    onClose: () => void
+}
+
+const BootstrapDialogTitle = (props: DialogTitleProps) => {
```

Comment:
Do you have any special style for this dialog? We have the public dialog component in @dimensiondev/maskbook-theme

Reply:
Updated the public dialog layout, and use MaskDialog here.

Reply:
Updated the public dialog layout, and use MaskDialog here.